<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C3DES396" ignored="0" name="TC_06_CompanyInternet_With_Addon_With_Wlan_Manual" tests="1" failures="1" timestamp="28 Feb 2018 15:44:48 GMT" time="468.425" errors="0">
  <testcase name="TestCaseExecution" time="457.539" classname="TestDriver.TestExecutionDriver">
    <failure type="java.lang.AssertionError" message=" failed">
      <![CDATA[java.lang.AssertionError:  failed
	at DUMMYDELETEIT.genericDriver.subOrdersValidationAndCompleteOrderInConsole(genericDriver.java:867)
	at TestDriver.TestExecutionDriver.TestCaseExecution(TestExecutionDriver.java:236)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- TestCaseExecution -->
</testsuite> <!-- TC_06_CompanyInternet_With_Addon_With_Wlan_Manual -->
